resources/build_resources.sh

```diff
@@ -13,6 +13,9 @@
 # The path to output all built .py files to: 
 UI_PYTHON_PATH=../python/tk_multi_publish/ui
 
+# The path to where the PySide binaries are installed
+PYTHON_BASE="/Applications/Shotgun.app/Contents/Resources/Python"
```

Comment:
I'm guessing we shouldn't hardcode this here. Should this be handled by the developer in whatever env they're working in? i.e. making sure `pyside-uic` and `pyside-rcc` are in their path? Maybe just a comment instead?